docs/shared/pdf_viewer.css

```diff
@@ -24,23 +24,18 @@
   line-height: 1.0;
 }
 
-.textLayer > div {
+.textLayer > span {
   color: transparent;
   position: absolute;
   white-space: pre;
   cursor: text;
-  -webkit-transform-origin: 0% 0%;
```

Comment:
I know the regular one is still in place, but the removal of these `transform-origin` styles, especially on the div itself may be what's causing the misalignment we keep seeing.

Reply:
Added in [this](https://github.com/atomicjolt/pdf-annotate.js/pull/1/commits/5d803a82c4e603886538b1b7858e31659e754ab1) commit